### ISSUE TITLE
[Requires Documentation] Add notes on how to export simulation for use on a renderfarm

### ISSUE THREAD

**Issue by rlguy** [rlguy/Blender-FLIP-Fluids#352]
There is a workaround for getting the FLIP Fluid cache to work on a render farm, but there is no documentation for this at the moment.

Notes:
- [Cache must be exported to Alembic](https://github.com/rlguy/Blender-FLIP-Fluids/wiki/Scene-Troubleshooting#export-to-alembic-cache-is-not-exporting-animated-meshes)
- For whitewater particles to render, you must use [dupliverts](https://docs.blender.org/manual/en/dev/editors/3dview/object/properties/duplication/dupliverts.html) over the whitewater meshes
- To access materials from the FLIP Fluids material library, you must import the materials (located in the 'FLIP Fluids Materials') panel

**Comment by JARINVER** [comment]
Hello,
I would like to follow up this topic. I am working on simulation of boat and I need to export it into external software (Unreal or any other 3D / render software). In best scenario as a full animation. Water with alembic works perfectly, but I still have a trouble with exporting of Foam as separate files also when i use dupliverts. After export they are invisible or in incorrect positions. 
I know, the article above told use modifier to help blender recognize whitewater (I think smooth modifier is as default), but it doesn´t work to me.
In short - how should I export water with foam in one file / of group of files which will be recognizable for external 3D software?
Thank you for you help.

**Comment by rlguy** [comment]
Hi JARINVER,

Thanks for sending us a message! See this expanded documentation topic that explains all of these points: [Alembic Export Suppport](https://github.com/rlguy/Blender-FLIP-Fluids/wiki/Alembic-Export-Support).

In short:
- I would suggest using the latest version of the FLIP Fluids addon with the latest version of Blender. There have been updates to Blender's Alembic exporter that improve stability and speed in later versions.
- Not all external 3D software may be compatible with Blender's Alembic structure. This is most common for meshes that contain only vertices. This is something that is outside of control at the moment.

Hope this info helps!

\- Ryan
